tests/providers/amazon/aws/operators/test_s3_file_transform.py

```diff
@@ -34,37 +29,33 @@
 from airflow.providers.amazon.aws.operators.s3 import S3FileTransformOperator
 
 
-class TestS3FileTransformOperator(unittest.TestCase):
-    def setUp(self):
+@pytest.fixture
+def transform_script_loc(request, tmp_path_factory):
+    transform_script = tmp_path_factory.mktemp(request.node.name) / "transform.py"
+    transform_script.touch()
+    yield str(transform_script)
```

Comment:
Create empty "transform" file by  `tmp_path_factory` template instead of manual creation temporary directory and clear up it